sds/src/proximity_keywords/mod.rs

```diff
@@ -527,27 +521,39 @@ mod test {
 
     #[test]
     fn excluded_keyword_should_have_word_boundaries() {
-        let (_included, excluded) =
-            try_new_compiled_proximity_keyword(30, vec![], vec!["host".to_string()]).unwrap();
-        let excluded = excluded.unwrap();
-
-        assert!(excluded.is_false_positive_match("host ping", 5));
-        assert!(!excluded.is_false_positive_match("localhost ping", 10));
-        assert!(!excluded.is_false_positive_match("hostlocal ping", 10));
-
-        // word boundaries are is added at the beginning (resp. end) only if the first (resp. last) character is a letter or a digit
-        let (_included, excluded) =
-            try_new_compiled_proximity_keyword(30, vec![], vec!["!host".to_string()]).unwrap();
-        let excluded = excluded.unwrap();
-        assert!(excluded.is_false_positive_match("!host- ping", 6));
-        assert!(excluded.is_false_positive_match("local!host ping", 11));
-        assert!(!excluded.is_false_positive_match("!hostlocal ping", 11));
-
-        let (_included, excluded) =
-            try_new_compiled_proximity_keyword(30, vec![], vec!["ৎhost".to_string()]).unwrap();
+        let (_included, excluded) = try_new_compiled_proximity_keyword(
+            30,
+            vec![],
+            vec![
+                String::from("host"),
```

Comment:
I think you only need a single excluded keyword to reproduce the issue (e.g. you can delete all but `host`)